From c95b9ded44c38dcde7b008cb0700533975765455 Mon Sep 17 00:00:00 2001
From: b07117 <b07117@freescale.com>
Date: Fri, 20 May 2011 00:24:28 +0800
Subject: [PATCH 150/635] ENGR00143661, Add property to switch
 auto-configuration feature

Need disable this feature for most displays, such as DVI/TVout

Signed-off-by: xianzhong <b07117@freescale.com>
---
 libgralloc/framebuffer.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/libgralloc/framebuffer.cpp b/libgralloc/framebuffer.cpp
index aa5b168..ede6c63 100755
--- a/libgralloc/framebuffer.cpp
+++ b/libgralloc/framebuffer.cpp
@@ -543,6 +543,10 @@ static int set_graphics_fb_mode(int fb)
     char fb_modes[256];
     char temp_name[256];
 
+    char value[PROPERTY_VALUE_MAX];
+    property_get("ro.AUTO_CONFIG_DISPLAY", value, "0");
+    if (strcmp(value, "1") != 0)  return 0;
+
     sprintf(temp_name, "/sys/class/graphics/fb%d/modes", fb);
     fp_modes = open(temp_name,O_RDONLY, 0);
     if(fp_modes < 0) {
-- 
1.8.0

